inventory/agents_mysqld_exporter_test.go

```diff
@@ -124,6 +124,59 @@ func TestMySQLdExporter(t *testing.T) {
 		}, changeMySQLdExporterOK.Payload)
 	})
 
+	t.Run("WithRealPMMAgent", func(t *testing.T) {
```

Comment:
Let's skip it for now

Reply:
I see that you are already skipping test here https://github.com/Percona-Lab/pmm-api-tests/pull/66/files#diff-c90a6ba65d81324ad9a343a83001667fR160, but let's skip it until we have pmm-agent in CI

Reply:
It is being skipped if there isn't a connected agent:
https://github.com/Percona-Lab/pmm-api-tests/blob/PMM-5026-Table_count_unknown/inventory/agents_mysqld_exporter_test.go#L159-L161

Reply:
yeah, I know, but let's skip it at all.